packages/conventional-changelog-odyssey/package.json

```diff
@@ -7,6 +7,6 @@
   "private": true,
   "sideEffects": false,
   "dependencies": {
-    "conventional-changelog-conventionalcommits": "^4.6.1"
+    "conventional-changelog-conventionalcommits": "^5.0.0"
```

Comment:
I'm betting this is the one that fixed it, but I upgraded other related packages too because I was already in here.